app/src/main/java/com/android/streetworkapp/utils/DateUtils.kt

```diff
@@ -73,13 +78,18 @@ fun dateDifference(endTimestamp: String): String {
 
   val hours = duration.toHours() % 24
 
+  val statusText =
+      when (event.status) {
+        EventStatus.STARTED -> "event has begun!"
```

Comment:
Typo: to stay coherent, perhaps the first letter should be capital, such as what has been done in line 85

(The same applies to the line 84)

In addition, the String.xml file could be used for such strings.